lib/index.d.ts

```diff
@@ -0,0 +1,65 @@
+import { Stats } from 'fs';
+
+export type CreateOptions = {
+  dot?: boolean;
+  globOptions?: {};
```

Comment:
Here should be a type for this? Options inside this object be any.

Reply:
That was an oversight, thanks for catching that. I'll need to add `@types/glob` to the dependencies.

Reply:
or possibly `optionalDependencies`?